services/data-info/src/data_info/services/manifest.clj

```diff
@@ -0,0 +1,79 @@
+(ns data-info.services.manifest
+  (:use [data-info.services.sharing :only [anon-file-url anon-readable?]]
+        [clj-jargon.init :only [with-jargon]]
+        [clj-jargon.metadata :only [get-attribute attribute?]]
+        [slingshot.slingshot :only [try+ throw+]])
+  (:require [clojure.tools.logging :as log]
+            [clojure-commons.file-utils :as ft]
+            [dire.core :refer [with-pre-hook! with-post-hook!]]
+            [data-info.util.validators :as validators]
+            [data-info.services.stat :as stat]
+            [data-info.services.uuids :as uuids]
+            [data-info.util.irods :as irods]
+            [data-info.util.logging :as dul]
+            [tree-urls-client.core :as tree]
+            [data-info.util.config :as cfg]))
+
+(def ^:private coge-attr "ipc-coge-link")
+
+;; this is usually going to be the vector case, but the previous behavior is the map case so it's preserved
+(defn- postprocess-tree-urls
+  [tree-urls]
+  (if (map? tree-urls)
+      (:tree-urls tree-urls)
+      (vec tree-urls)))
```

Comment:
This function is new. Tree-urls, at some point, stopped returning `{"tree-urls": [list of urls]}` and switched to just returning `[list of urls]`, so the manifests haven't been getting this. It appeared "correctly" in some sense because you have to mess with info-types to be able to produce extra visualization URLs to screw it up (files to which tree URLs apply can't be sent to coge [not that coge's AVU adding works right now anyway] or genome browser [i.e. anon-files]), so it got an empty string and the UI then tried to immediately generate a tree URL. But having it actually parse it correctly is probably a good idea.